src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs

```diff
@@ -164,7 +164,7 @@ public void GetExecutablePath_should_find_existing_executable()
             [Fact]
             public void GetExecutablePath_should_find_existing_executable_with_extension()
             {
-                FileSystem.get_executable_path("cmd.exe").ShouldEqual(
+                FileSystem.get_executable_path("cmd.exe").ToLower().ShouldEqual(

```

Comment:
I've seen \Windows and \windows returned, I guess it depends on how OS was installed...